codegen-on-oss/codegen_on_oss/analysis/symbolattr.py

```
@@ -30,7 +30,9 @@ def print_symbol_attribution(codebase):
 
     # Print attribution for top symbols
     count = 0
-    for symbol, usage_count in symbols_with_usages[:10]:  # Look at top 10 most used symbols
+    for symbol, usage_count in symbols_with_usages[
+        :10
+    ]:  # Look at top 10 most used symbols
         count += 1
```

Ruff (0.8.2)

Use `enumerate()` for index variable `count` in `for` loop

(SIM113)